lib/fat_zebra/utilities/apple_pay/domain.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module FatZebra
+  module Utilities
+    module ApplePay
+      class Domain < APIResource
+
+        ENDPOINT_URL = '/v1.0/utilities/apple_pay/domains'
+
+        class << self
+
+          ##
+          # Register an Apple Pay (web) domain
+          #
+          # @return [FatZebra::Utilities::ApplePay::Domains] response
+          def register!(domain, params = {})
+            response = request(:post, path(domain), params)
```

Comment:
The gateway endpoints don't seem to validate the given domain in any way. Should I add that validation here?

Reply:
Apple will happily decline any invalid value